janusgraph-core/src/main/java/org/janusgraph/graphdb/tinkerpop/optimize/strategy/AbstractJanusGraphMixedIndexAggStrategy.java

```diff
@@ -47,6 +50,13 @@ public void apply(final Traversal.Admin<?, ?> traversal) {
         });
     }
 
+    @Override
+    public Set<Class<? extends ProviderOptimizationStrategy>> applyPost() {
+        Set<Class<? extends ProviderOptimizationStrategy>> postStrategies = new HashSet<>();
+        postStrategies.add(JanusGraphLocalQueryOptimizerStrategy.class);
```

Comment:
nitpick: we can use a static final variable with value `Collections.singleton(JanusGraphLocalQueryOptimizerStrategy.class)`.

Reply:
Good suggestion, I also changed it in a different strategy from which I copied this implementation.

Reply:
I noticed this as well but I checked that in every JanusGraph strategy we use this approach. I don't know exactly why we don't use constant in JanusGraph strategies here.
Nevertheless, I think it would make sense to change that in each strategy and not only in the current strategy. Thus, I think it's OK to do it in a separate PR

Reply:
Ah, I think I misunderstood your suggestion a bit, @li-boxuan. I just changed the implementation to use `Collections.singleton()` directly here, but didn't add a final variable. Let me add that and I'll also do it for the other strategies.
If we're already discussing this here, then I think it's the easiest if we also implement it here so you can directly review it.

Reply:
done